providers/os/resources/services/manager.go

```diff
@@ -157,7 +157,9 @@ func ResolveManager(conn shared.Connection) (OSServiceManager, error) {
 	case asset.Platform.Name == "windows":
 		osm = &WindowsServiceManager{conn: conn}
 	case asset.Platform.Name == "alpine":
-		osm = &AlpineOpenrcServiceManager{conn: conn}
+		osm = &OpenrcServiceManager{conn: conn}
+	case asset.Platform.Name == "gentoo":
```

Comment:
This will not work in all cases since Gentoo also supports systemd, we need to detect the selected init https://wiki.gentoo.org/wiki/Systemd and https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/System#Init_and_boot_configuration

Reply:
Ok to merge as it does not break anything, but we need to add support to detect systemd services on Gentoo when it is configured.